### PR TITLE
Add rustlings repository under automation

### PR DESCRIPTION
**Pull request by Kobzol**
Repo: https://github.com/rust-lang/rustlings

CC @shadows-withal

Not sure if there should be a separate team for this?

Extracted from GH:
```toml
org = "rust-lang"
name = "rustlings"
description = ":crab: Small exercises to get you used to reading and writing Rust code!"
bots = []

[access.teams]
security = "pull"

[access.individuals]
jdno = "admin"
shadows-withal = "admin"
rust-lang-owner = "admin"
pietroalbini = "admin"
rylev = "admin"
Mark-Simulacrum = "admin"
AbdouSeck = "write"
badboy = "admin"
jrvidal = "write"
carols10cents = "write"
```

### CODE REVIEW

File: repos/rust-lang/rustlings.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+org = "rust-lang"
+name = "rustlings"
+description = ":crab: Small exercises to get you used to reading and writing Rust code!"
+bots = []
+
+[access.teams]
+
+[access.individuals]
+# This repository probably needs a separate team?
+shadows-withal = "write"
```
Comment (rylev) [inline repos/rust-lang/rustlings.toml:10]: I wonder if we should just do this as part of this PR?

Comment (Kobzol) [reply repos/rust-lang/rustlings.toml:10]: It seems like sometimes we create teams just on a whim, but sometimes it requires an RFC? (https://rust-lang.zulipchat.com/#narrow/stream/301329-t-devtools/topic/mdbook.20team). I'm not sure what's the proper way.

Comment (shadows-withal) [reply repos/rust-lang/rustlings.toml:10]: yeah, I've thought about this before, but I'm not even sure where a "team" here would fit under? language? infra? devtools?

**Review by shadows-withal — APPROVED** [review]

**Review by Mark-Simulacrum — APPROVED** [review]
Going to go ahead and merge so at least the status-quo is documented & maintained by automation.

**Pull request merged**